pipeline_dp/pipeline_operations.py

```diff
@@ -124,52 +123,45 @@ def group_by_key(self, col, stage_name: str):
     def filter(self, col, fn, stage_name: str):
         return col | stage_name >> beam.Filter(fn)
 
-    def filter_by_key(self, col, keys_to_keep, data_extractors,
-                      stage_name: str):
+    def filter_by_key(self, col, keys_to_keep, stage_name: str):
 
         class PartitionsFilterJoin(beam.DoFn):
```

Comment:
Curious, my IDE complains that this method isn't implemented from the abstract super class, not sure if it's needed?

```
           def to_runner_api_parameter(self, unused_context):
                pass
```

Reply:
It's probably some error of parsing by IDE. This method is not abstract. 

And according to [example](https://beam.apache.org/documentation/transforms/python/elementwise/pardo/), it's needed only to implement `process()`